util/metrics/src/service_metrics.rs

```diff
@@ -95,7 +95,9 @@ impl ServiceMetrics {
             .unwrap(),
         }
     }
+}
 
+impl ServiceMetrics {
```

Comment:
Why split this impl block? I don't see a Clippy lint for this or anything (actually there's a negative Clippy lint but it's restricted)

Reply:
And actually shouldn't the upper impl block (line 68) be `impl Default for ServiceMetrics`?

Reply:
Yes, you're right. Done.